Components/Card/card.css

```diff
@@ -0,0 +1,83 @@
+.center-block.card-ex {
+  width: 70%;
+}
+
+.card {
+  padding: 10px;
+  height: fit-content;
+  background-color: var(--yellow-color-light);
+  border-radius: 7px;
+}
+
+.hor-card {
+  width: 100%;
+  display: grid;
+  grid-template-columns: 200px 1fr;
+  align-items: center;
+}
+
+.ver-card {
+  width: 150px;
+  min-width: fit-content;
+  height: fit-content;
+  display: grid;
+  grid-template-rows: 200px auto;
+}
+
+.var-card .img-container {
+  height: 100%;
+  width: 100%;
+}
+
+.hor-card .img-container {
+  height: 100%;
+  width: 170px;
+  margin-right: 30px;
+}
+
+.text-card {
+  height: 90%;
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: flex-start;
+}
+
+.img-container img {
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+}
+
+.money {
+  display: flex;
+  align-items: center;
+  font-size: 1.5rem;
+}
```

Comment:
This naming for the card money is a bit vague, can be changed to something like card-money, ver-card-money or hor-card-money

Reply:
okay will do that